Resources/Prototypes/Voice/speech_emote_sounds.yml

```diff
@@ -88,6 +88,8 @@
       collection: MaleCry
     Weh:
       collection: Weh
+    WehChat:
+      collection: WehChat
 
```

Comment:
This doesn't need to be its own collection.

Reply:
I am just a new contributor, and I didn't find a way how I can make lizards only weh. Cuz adding chat triggers on normal weh will make it so anyone else will be able to weh. And I have to keep it because of that the "juice that makes you weh". So I created a new one for lizards to use only that will not be called when they metabolised weh juice. But I can try making it in one thing. Hopefully this explanation is appropriate, I am not a native English speaker and is new in contribution to this game.

Reply:
i can make it so only lizards will be able to weh, and it will be that so no other species can weh. But it will make weh juice only working on lizards. Only thing it will be able to do to any other species is make them become lizard for some time. 
But it will be in one collection.